scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -276,6 +301,242 @@ private static void unzipReportResources(InputStream source, File target) throws
         }
     }
 
+    /**
+     * Returns the {@link ScanTomlFile} representation of a local/remote Scan.toml file.
+     *
+     * @param project      Ballerina project
+     * @param outputStream print stream
+     * @return an in-memory representation of the Scan.toml file
+     */
+    public static ScanTomlFile loadScanTomlConfigurations(Project project, PrintStream outputStream) {
+        Path targetDir = project.targetDir();
+
+        if (project.kind().equals(ProjectKind.BUILD_PROJECT)) {
+            if (project.currentPackage().ballerinaToml().isPresent()) {
+                BallerinaToml ballerinaToml = project.currentPackage().ballerinaToml().get();
+                TomlDocument ballerinaTomlDocument = ballerinaToml.tomlDocument();
+                Toml ballerinaTomlDocumentContent = ballerinaTomlDocument.toml();
+                Toml scanTable = ballerinaTomlDocumentContent.getTable(SCAN_TABLE).orElse(null);
+
+                if (scanTable != null) {
+                    TomlValueNode configPath = scanTable.get(SCAN_FILE_FIELD).orElse(null);
+
+                    if (configPath != null) {
```

Comment:
Check the inverse and return early. That way we can reduce the indentation for the larger part of the logic.